tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -167,3 +168,57 @@ describe('People And Roles component', () => {
     resetStore()
   })
 })
+
```

Comment:
Added a few tests for Short amalgamation (it is a bit of a different case than other filings)